examples/tools/go_db.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+import pyscf
+from pyscf.tools.go_db import get_xyz
+import requests
+
+if __name__ == '__main__':
+    VERBOSITY = 9
+    CONV_TOL = 1e-7
+    BASIS = 'sto-3g'
+    FROZEN = 4
```

Comment:
VERBOSITY, CONV_TOL, BASIS, and FROZEN are not necessary for an example calculation. Especially when no post-HF calculation is being done (so `frozen` is meaningless) and when no HF calculation is done (so even `conv_tol` is meaningless). I've kept the `basis` and `verbosity` so that users can see that they can change these if they want, but I've just done this in the argument to `pyscf.M`. 

For production-level calculations I agree that it's better to define some of these parameters at the beginning rather than "hard-coding" them into the function calls, but  for an example, in which changing these parameters has nothing to do with the feature being exemplified, I think it's better to keep the example file simpler by just hard-coding them.

I've already made this change in my latest commit.

Reply:
Agreed.